powershell-gallery/docs-conceptual/powershellget/install-powershellget.md

```diff
@@ -0,0 +1,54 @@
+---
+description: This article explains how install PowerShellGet.
+ms.date: 04/07/2023
+title: How to Install PowerShellGet
+---
+# How to Install PowerShellGet
+
+## Prerequisites
+
+Ensure that you have a version of **PowerShellGet** and **PackageManagement** newer than 1.0.0.1
+installed. The latest stable version is **PowerShellGet** 2.2.5 with **PackageManagement** is
+1.4.8.1.
```

Comment:
I think `is` should be replaced. Keeping the structure as similar as possible, perhaps:

```suggestion
installed. The latest stable version is **PowerShellGet** 2.2.5 with **PackageManagement** at
1.4.8.1.
```

Alternatively:

```suggestion
installed. The latest stable versions are 2.2.5 for **PowerShellGet** and 1.4.8.1 for
**PackageManagement**.
```